tools/config_updater.py

```diff
@@ -553,26 +582,29 @@ def main(unused_argv):
     ShowUser(flags.FLAGS.username, token=token)
 
   elif flags.FLAGS.subparser_name == "update_user":
-    password = None
-    if flags.FLAGS.password:
-      password = getpass.getpass(prompt="Please enter new password for '%s': " %
-                                 flags.FLAGS.username)
-    UpdateUser(flags.FLAGS.username, password, flags.FLAGS.add_labels,
-               flags.FLAGS.delete_labels, token=token)
+    try:
+      UpdateUser(flags.FLAGS.username, flags.FLAGS.password, flags.FLAGS.add_labels,
+                 flags.FLAGS.delete_labels, token=token)
+    except (aff4.InstantiationError):
+      print "User %s does not exist." % flags.FLAGS.username
 
   elif flags.FLAGS.subparser_name == "delete_user":
     DeleteUser(flags.FLAGS.username, token=token)
 
   elif flags.FLAGS.subparser_name == "add_user":
-    if flags.FLAGS.password:
-      password = flags.FLAGS.password
-    else:
-      password = getpass.getpass(
-          prompt="Please enter password for user '%s': " % flags.FLAGS.username)
     labels = []
     if not flags.FLAGS.noadmin:
       labels.append("admin")
-    UpdateUser(flags.FLAGS.username, password, labels, token=token)
+
+    if flags.FLAGS.labels:
+      labels.extend(flags.FLAGS.labels)
+
+    try:
+      if aff4.FACTORY.Open("aff4:/users/%s" % flags.FLAGS.username, "GRRUser",
```

Comment:
We should have the checks inside the functions and have them print meaningful errors. Would just give a better user experience. For the init, maybe we can just test if the user already exists? If we did, we could maybe even go a step further and ask if we should change the password, in most cases I don't really want to do that anyways. What do you think?